### PR TITLE
amazonbidding/add_podspecs

### PR DESCRIPTION
**Pull request by Lorenzo45**
https://app.asana.com/0/1128417577549588/1131962640981678

### CODE REVIEW

File: AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+s.authors = 
+{
+    'a9' => 'a9-pdp-dev@amazon.com',
+    'Dev Services' => 'devservices-mobile-sdk@amazon.com'
+}
+s.name = 'AmazonBiddingSDK'
+s.version = '2.2.17.0'
+s.platform = :ios
+s.ios.deployment_target = '8.0'
```
Comment (thomasmso) [inline AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec]: Do you know what's their min iOS version?

File: AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+{ 
+    :http => 'https://bintray.com/applovin/iOS-Adapter-SDKs/download_file?file_path=AmazonBiddingSDK-2.2.17.0.zip',
+    :type => 'zip',
+    :sha256 => '6480563cc03dee604d74a00c4880a6458533f993b8b32a873991c2a83259c83b'
+}
+s.description = <<-DESC
+	Monetize your apps across platforms with great eCPM and high-quality ads from Amazon and brand advertisers.
+DESC
+
+s.frameworks = 'AdSupport', 'CoreLocation', 'SystemConfiguration', 'CoreTelephony', 'MediaPlayer', 'EventKit', 'EventKitUI', 'StoreKit', 'QuartzCore', 'SafariServices', 'JavaScriptCore'
```
Comment (thomasmso) [inline AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec]: Whaaa they require CoreLocation?? And EventKit (your cal)?? I'll hit up Idil on the chat to see what's up

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+The AppLovin MAX SDK is available under a commercial license (https://www.applovin.com/eula).
+
+LICENSE
+}
+
+s.source = 
+{
+    :http => 'https://bintray.com/applovin/iOS-Adapters/download_file?file_path=AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0.zip',
+    :type => 'zip',
+    :sha256 => '3c922c3275fc8f44e843a19f6ce3c429a2991c74475bbc92029a3a627fb4809c'
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
    :sha256 => 'SHA_PLACEHOLDER'
```

File: AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec (hunk 1 above)
Comment (Lorenzo45) [reply AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec]: Actually checked their info.plist and it's 9.0. Updated

File: AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec (hunk 2 above)
Comment (Lorenzo45) [reply AmazonBiddingSDK/2.2.17.0/AmazonBiddingSDK.podspec]: Pending email confirmation but removed it for now.

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The AppLovin MAX SDK is available under a commercial license (https://www.applovin.com/eula).
+
+LICENSE
+}
+
+s.source = 
+{
+    :http => 'https://bintray.com/applovin/iOS-Adapters/download_file?file_path=AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0.zip',
+    :type => 'zip',
+    :sha256 => 'SHA_PLACEHOLDER'
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
```

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+s.version = '2.2.17.0.0'
+s.platform = :ios, '9.0'
+s.summary = 'Amazon Bidding adapter used for mediation with the AppLovin MAX SDK'
+s.homepage = 'https://www.applovin.com/'
+s.license = 
+{ 
+    :type => 'Commercial License',
+    :text => <<-LICENSE
+
+Copyright 2018 AppLovin Corp. All rights reserved.
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
Copyright 2019 AppLovin Corp. All rights reserved.
```

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+s.source = 
+{
+    :http => 'https://bintray.com/applovin/iOS-Adapters/download_file?file_path=AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0.zip',
+    :type => 'zip',
+}
+
+s.vendored_frameworks = 'AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.framework'
+
+s.dependency 'AmazonBidding', '= 2.2.17.0'
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
s.dependency 'AmazonBiddingSDK', '= 2.2.17.0'
```

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 7]
```diff
@@ -0,0 +1,49 @@
+Pod::Spec.new do |s|
+
+s.authors =
+{
+    'AppLovin Corporation' => 'support@applovin.com'
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
    'AppLovin Corporation' => 'devsupport@applovin.com'
```

File: AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+s.source = 
+{
+    :http => 'https://bintray.com/applovin/iOS-Adapters/download_file?file_path=AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0.zip',
+    :type => 'zip',
+}
+
+s.vendored_frameworks = 'AppLovinMediationAmazonBiddingAdapter-2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.framework'
+
+s.dependency 'AmazonBiddingSDK', '= 2.2.17.0'
+s.dependency 'AppLovinSDK', '>= 6.0.0'
```
Comment (thomasmso) [inline AppLovinMediationAmazonBiddingAdapter/2.2.17.0.0/AppLovinMediationAmazonBiddingAdapter.podspec]:
```suggestion
s.dependency 'AppLovinSDK'
```

**Pull request merged**